jenkins/deduper.py

```diff
@@ -20,6 +21,7 @@
 
 
 class GhprbOutdatedBuildAborter:
+
```

Comment:
I use the sublime plugin and it automatically added these lines when I was working on these files before.